tests/test_A000584.py

```diff
@@ -0,0 +1,4 @@
+from oeis import A000584
+
+def test_A000584():
+	assert A000584[:20] == [0, 1, 32, 243, 1024, 3125, 7776, 16807, 32768, 59049, 100000, 161051, 248832, 371293, 537824, 759375, 1048576, 1419857, 1889568, 2476099]
```

Comment:
You could add another test for this one, using hypothesis, like:

```python
@given(integers())
def test_A000584_mod(i):
    assert A000584[i] % 10 == i % 10
```

more generally, when sequences have comments that could be implemented as tests, I like to implement them.

Often though we have to put limits on the `integers()` strategy, like `min_value=0`, but also maybe a `max_value`, which has the nice property to "discover"/"document" the limits of our implementation.